### PR TITLE
Microseconds Everywhere - CTable

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

**Comment by czue** [comment]
build failing

File: ctable/models.py [hunk 1]
```diff
@@ -100,7 +110,12 @@ def convert_type(self, value):
 
         try:
             if self.data_type == "date" or self.data_type == "datetime":
-                converted = datetime.strptime(value, self.date_format or "%Y-%m-%dT%H:%M:%SZ")
+                if self.date_format:
+                    converted = datetime.strptime(value, self.date_format)
+                else:
+                    # I'd normally use iso_string_to_datetime,
+                    # but that's in corehq which isn't shared by this repo
+                    converted = DateTimeProperty().wrap(value)
```
Comment (dannyroberts) [inline ctable/models.py]: @snopoke assuming tests pass, let me know what you think of this solution

Comment (snopoke) [inline ctable/models.py]: :+1:

**Comment by snopoke** [comment]
Assuming this shouldn't be merged yet (added Don't merge label)

**Comment by czue** [comment]
:white_check_mark: to merge

**Pull request merged**